http/README.md

```diff
@@ -278,3 +278,19 @@ deleteCookie(headers, "deno", { path: "/", domain: "deno.land" });
 ```
 
 > Note: At the moment multiple `Set-Cookie` in a `Response` is not handled.
+
+### getSetCookies
```

Comment:
I think this API is for client side handling of cookies (This is something you use when you're interacting server from client side)

Can you describe a little more about it as other methods are for server side handling of cookies, and users might be confused about the purpose of this API.

Reply:
Yes, this is for client-side cookie handling. I can't think of any other context in parsing the `Set-Cookie` header that would be needed besides on the client.

Again, [Go's cookie.go](https://github.com/golang/go/blob/master/src/net/http/cookie.go) has the alike `readSetCookies` function, and there's a [comment in the current code](https://github.com/denoland/deno_std/blob/cac7764e3878ac8c2a3c26a1201f3dfcc15e6910/http/cookie.ts#L194-L195) that points out that such a function is needed (if I read it right).

True, it is the only client-side-based function here. But I think it is useful, and related functionality appears to be wanted by some.